probe/docker/container.go

```diff
@@ -22,11 +22,12 @@ import (
 
 // These constants are keys used in node metadata
 const (
-	ContainerName    = "docker_container_name"
-	ContainerCommand = "docker_container_command"
-	ContainerPorts   = "docker_container_ports"
-	ContainerCreated = "docker_container_created"
-	ContainerIPs     = "docker_container_ips"
+	ContainerName        = "docker_container_name"
+	ContainerCommand     = "docker_container_command"
+	ContainerPorts       = "docker_container_ports"
+	ContainerCreated     = "docker_container_created"
+	ContainerIPs         = "docker_container_ips"
+	ContainerLabelPrefix = "docker_container_label_"
```

Comment:
Which I think you forgot to add?

On Wed, Aug 26, 2015 at 6:09 PM, Alfonso Acosta <notifications@github.com>
wrote:

> In probe/docker/container.go
> <https://github.com/weaveworks/scope/pull/400#discussion_r38007572>:
>
> > @@ -22,11 +22,12 @@ import (
> >
> >  // These constants are keys used in node metadata
> >  const (
> > -	ContainerName    = "docker_container_name"
> > -	ContainerCommand = "docker_container_command"
> > -	ContainerPorts   = "docker_container_ports"
> > -	ContainerCreated = "docker_container_created"
> > -	ContainerIPs     = "docker_container_ips"
> > +	ContainerName        = "docker_container_name"
> > +	ContainerCommand     = "docker_container_command"
> > +	ContainerPorts       = "docker_container_ports"
> > +	ContainerCreated     = "docker_container_created"
> > +	ContainerIPs         = "docker_container_ips"
> > +	ContainerLabelPrefix = "docker_container_label_"
>
> Oh, thanks, that shouldn't even be there anymore, there's a new label.go
> file for this.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/weaveworks/scope/pull/400/files#r38007572>.
>